packages/good-design/src/core/layout/ClaimCarousel.tsx

```diff
@@ -27,7 +27,7 @@ const SlidesComponent = memo(({ activeSlide, slidesNumber }: { activeSlide: numb
     {Array(slidesNumber)
       .fill(0)
       .map((_, index, arr) => (
-        <SlideMark key={index} isActive={index === activeSlide} isLast={index === arr.length - 1} />
+        <SlideMark key={_} isActive={index === activeSlide} isLast={index === arr.length - 1} />
```

Comment:
number of slide is the same index,
some unique value describing slide should be used

for example "what-is-g$", "how-to-claim", "why-i-should-fv" etc 
e.g. **depending on slide contents**

Reply:
done